aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -214,9 +159,18 @@ SparseTensor new_with_tensor_and_size_sparse(const LongTensor& indices, const Te
   AT_CHECK(sizes.size() == sparseDims + denseDims, "number of dimensions must be sparseDims (", sparseDims, ") + denseDims (", denseDims, "), but got ", sizes);
 
   LongTensor max_indices = std::get</* values */ 0>(indices.max(/* dim */ 1, /* keepdim */ false));
-  auto max_indices_accessor = max_indices.accessor<int64_t, 1>();
+  LongTensor cpu_max_indices;
+  if (max_indices.is_cuda()) {
+    cpu_max_indices = at::CPU(kLong).tensor(max_indices.sizes());
```

Comment:
nit: `cpu_max_indices = at::CPU(kLong).copy(max_indices)` ?

Reply:
Sweet! Didn't know I could do that.